entrypoint/selinux/upgradeselinux_test.go

```diff
@@ -0,0 +1,99 @@
+//go:build upgradeselinux
```

Comment:
on this context thus file makes no much value!

you should on the selinux test use like

	if customflag.ServiceFlag.InstallMode.String() != "" { YOUR UPGRADE FUNTIONS }